### PR TITLE
Introduce error notification for source view

### PR DESCRIPTION
**Pull request by prabushi**
## Purpose
Fix https://github.com/wso2/product-ei/issues/2466

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/DummyEndpointFactory.java [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+
+import javax.xml.namespace.QName;
+
+
+public abstract class DummyEndpointFactory {
+
+	private DefinitionFactory customDefnFactory = null;
```
Comment (dakshika) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/DummyEndpointFactory.java]: formatting issue

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/validator/ProcessSourceView.java [hunk 2]
```diff
@@ -0,0 +1,1041 @@
...
+
+		xmlTag.setLine(index + 1);
+
+		if (hasTag(value)) {
+			// has a tag
+			while (hasTag(value) || hasString(value)) {
+				xmlTag = new XMLTag();
+				xmlTag.setLine(index + 1);
+
+				int sT = value.indexOf("<");
```
Comment (indikasampath2000) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/validator/ProcessSourceView.java]: sT and eT should have a proper name.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/validator/ProcessSourceView.java [hunk 3]
```diff
@@ -0,0 +1,1041 @@
...
+				int eT = value.indexOf(">");
```
Comment (indikasampath2000) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/validator/ProcessSourceView.java]: sT and eT should have a proper name.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/DummyEndpointFactory.java (hunk 1 above)
Comment (prabushi) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/DummyEndpointFactory.java]: Updated.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/validator/ProcessSourceView.java (hunk 3 above)
Comment (prabushi) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/validator/ProcessSourceView.java]: Updated.

**Pull request merged**